src/mt_nanopb.c

```diff
@@ -0,0 +1,15 @@
+/* Automatically generated nanopb constant definitions */
+/* Generated by nanopb-0.4.6-dev */
+
+#include "mt_nanopb.h"
+
+#if PB_PROTO_HEADER_VERSION != 40
+#error Regenerate this file with the current version of nanopb generator.
+#endif
+
+PB_BIND(User, User, AUTO)
+PB_BIND(Position, Position, AUTO)
+PB_BIND(NodeInfo, NodeInfo, AUTO)
+PB_BIND(MyNodeInfo, MyNodeInfo, AUTO)
+PB_BIND(FromRadio, FromRadio, AUTO)
+PB_BIND(ToRadio, ToRadio, 2)
```

Comment:
I'd try to make the protobuf generated code dynamically generated, via ci or local build script & CI, less stuff to check in and whatnot - and dependency management can be as simple as referencing the desired commit via submodules.

Reply:
That's not really the Arduino way; you're supposed to just be able to install a library, select an example, and click Upload. A big part of building an IoT MT device is going to be deciding which subset of mesh.proto to include; bringing in the whole thing is probably too much for a lot of embedded projects.

That said, a tool to help out with this process would be great, but the Arduino Library Repository wouldn't really be the right place for it.